pyproject.toml

```diff
@@ -0,0 +1,91 @@
+[tool.black]
+target-version = ['py311']
+
+# This pyproject.toml is setup so it can be used with or without poetry and also
+# supports editable installs (PEP 660) without breaking IDE and linter inspection.
+
+# DEVELOPER INSTALL INSTRUCTIONS (assuming current directory is at the top of the project) :
+# Poetry:
+#   poetry install --extras dev_all
+# PIP:
+#   pip install -e .[dev_all]
+
+[tool.poetry]
+name = "screenpy_adapter_allure"
+version = "4.0.2"
+description = "An adapter for the microphone of ScreenPy's Narrator which sends logs to Allure."
+authors = ["Perry Goy <perry.goy@gmail.com>"]
+maintainers = ["Gabe Langton", "Marcel Wilson"]
```

Comment:
Hey @Scot3004, wanna add your name to this list? 😉

Reply:
Yes 😄